LBH.AdultSocialCare.Api/V1/Gateways/Payments/Concrete/PayRunInvoiceGateway.cs

```diff
@@ -116,6 +116,12 @@ public async Task<decimal> GetPayRunInvoicedTotalAsync(Guid payRunId)
             return decimal.Round(result, 2);
         }
 
+        public async Task<int> GetReleasedInvoiceCountAsync()
+        {
+            return await _dbContext.PayrunInvoices.Where(pi => pi.InvoiceStatus == InvoiceStatus.Released)
```

Comment:
What is the released status? Where did you see? InvoiceStatus can only be `Approved, Reject, Held`

So, you should use `Held` Also, you should check payrun status. In your query, you are counting for archived Payrun. 

`.Where(pi => pi.InvoiceStatus == InvoiceStatus.Held && pi.Payrun.Status != Archived)`

Reply:
Pretty sure that's what happens in held payments tab to get released holds. held invoice -> released -> into released holds pay run. No?
![image](https://user-images.githubusercontent.com/81692554/141319603-c1112315-b77c-4b81-abe2-a16704db2e23.png)

Reply:
Done